### PR TITLE
feat(@wallet): add all account action

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml [hunk 1]
```diff
@@ -267,8 +223,10 @@ Rectangle {
                         accountContextMenu.item.popup(mouse.x, mouse.y)
                         return
                     }
+                    console.log(model.address)
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/views/LeftTabView.qml:226]: left over?

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml [hunk 2]
```diff
@@ -321,6 +279,90 @@ Rectangle {
 
             readonly property bool footerOverlayed: contentHeight > availableHeight
 
+            header: AbstractButton {
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/views/LeftTabView.qml:282]: Control would have been just fine, I am not sure if AbstractButton if this is helping you in anyway?
Also can we move this to separate qml file under controls? its become very long.

**Review by Khushboo-dev-cpp — COMMENTED** [review]
Hey looks good, just some question on "All accounts" button

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml [hunk 3]
```diff
@@ -321,6 +279,90 @@ Rectangle {
...
+                id: header
+                verticalPadding: Style.current.padding
+                horizontalPadding: Style.current.padding
+
+                property bool highlighted: true
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/views/LeftTabView.qml:287]: `Button` already has this property btw

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml [hunk 4]
```diff
@@ -321,6 +279,90 @@ Rectangle {
...
+                }
+                
+                background: Rectangle {
+                    MouseArea {
+                        id: mouseArea
+                        anchors.fill: parent
+                        acceptedButtons: Qt.RightButton
+                        cursorShape: Qt.PointingHandCursor
+                        onClicked: {
+                            mouse.accepted = true
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/views/LeftTabView.qml:304]: This doesn't do anything

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml (hunk 2 above)
Comment (caybro) [reply ui/app/AppLayouts/Wallet/views/LeftTabView.qml:282]: Seconded

**Review by stefandunca — APPROVED** [review]
It looks good. Please consider all the comments.

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml (hunk 2 above)
Comment (alaibe) [reply ui/app/AppLayouts/Wallet/views/LeftTabView.qml]: Switch from abstract button to control. However when I extract to a component, thing just don't work in terms of layout
Also parent.ListView.view becomes undefined for no obvious reason to me

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml (hunk 3 above)
Comment (alaibe) [reply ui/app/AppLayouts/Wallet/views/LeftTabView.qml:286]: used button

File: src/app/modules/main/wallet_section/filter.nim [hunk 5]
```diff
@@ -24,9 +25,19 @@ proc `$`*(self: Filter): string =
     )"""
 
 
+proc setFillterAllAddresses*(self: Filter) = 
+  self.addresses = self.controller.getWalletAccounts().map(a => a.address)
+
+proc toggleWatchOnlyAccounts*(self: Filter) =
+  self.excludeWatchOnly = not self.excludeWatchOnly
```
Comment (saledjenic) [inline src/app/modules/main/wallet_section/filter.nim:32]: @alaibe maybe you have an idea how to do it, but just want to share some thoughts on this. 

A request is that user should be able to include/exclude watch only accounts in total balance from the settings and as well from the wallet section. Also that's the setting which need to be stored and that maintained per user, not per the app installation.

Now based on that fact I don't see filter as a good place for maintaining `excludeWatchOnly` value.

It should be stored in let's say new table `accounts_settings` (where we will store settings per account (having columns `key_uid`, `entity`, `string_value`, `int_value`, `bool_value`, defining `key_uid` as foreign key which  references `accounts.key_uid`)).

Then in wallet service we need to have a function `toggleWatchOnlyAccountsInBalances` which will toggle the value in db first and then send signal `INCLUDE_WO_ACC_IN_BALANCE_CHANGED` sending true or false as payload. 

Then we need to connect to that signal in wallet section and wallet settings. When we receive that signal in wallet section, from there we need to update filter calling `self.filter.includingWatchOnlyAccountsInBalancesUpdated(arg.value)`.

**Review by saledjenic — APPROVED** [review]

File: src/app/modules/main/wallet_section/filter.nim (hunk 5 above)
Comment (alaibe) [reply src/app/modules/main/wallet_section/filter.nim:32]: I merge this one and work on persist it

**Pull request merged**